src/software/simulated_tests/simulated_test_fixture.cpp

```diff
@@ -60,6 +60,11 @@ void SimulatedTestFixture::SetUp()
         enableVisualizer();
     }
     setupReplayLogging();
+
+    total_tick_duration = 0.0;
+    max_tick_duration   = 0.0;
+    min_tick_duration   = std::numeric_limits<double>::max();
```

Comment:
Umm I am a bit confused why `min_tick_duration` is at the maximum of double at `max_tick_duration` is 0

Reply:
this is the initial value so when the actual tick durations are registered then they can be compared against 0.0 for max_tick_duration and max double for min tick duration